boilerplate/flyte/docker_build/Readme.rst

```diff
@@ -11,13 +11,13 @@ If git head has a git tag, the Dockerhub image will also be tagged ``<IMAGE>:<GI
 
 **To Enable:**
 
-Add ``lyft/docker_build`` to your ``boilerplate/update.cfg`` file.
+Add ``flyteorg/docker_build`` to your ``boilerplate/update.cfg`` file.
 
-Add ``include boilerplate/lyft/docker_build/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
+Add ``include boilerplate/flyte/docker_build/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
```

Comment:
should this be flyteorg?
```suggestion
Add ``include boilerplate/flyteorg/docker_build/Makefile`` in your main ``Makefile`` _after_ your REPOSITORY environment variable
```

Reply:
yes, but I just use the same names that I found in the boilerplate.

Reply:
it's just `flyte` for now - my vote is to leave this as is for now, can change in the future.

Reply:
https://github.com/flyteorg/boilerplate/tree/master/boilerplate/flyte